test/runtests.jl

```diff
@@ -50,7 +50,9 @@ if anyerrors
     throw("Tests failed")
 end
 
-stdin = joinpath(dirname(@__FILE__), "stdin.sh")
-ENV2 = copy(ENV)
-ENV2["JULIA_HOME"] = JULIA_HOME
-run(setenv(`bash $stdin`, ENV2))
+if !is_windows()
```

Comment:
Perhaps StatsBase could provide an abstraction for arrays with missing data and/or categorical values, then StatsModels could be written against that, assuming NullableArrays and DataArrays can be made to adhere to whatever common interface would be required for such an abstraction. Or we can just copy some of the code from StatsModels over...